docs/config/README.md

```diff
@@ -1591,13 +1590,14 @@ The `hostname` module shows the system hostname.
 
 ### Options
 
-| Option     | Default                     | Description                                                                                                                          |
-| ---------- | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
-| `ssh_only` | `true`                      | Only show hostname when connected to an SSH session.                                                                                 |
-| `trim_at`  | `"."`                       | String that the hostname is cut off at, after the first match. `"."` will stop after the first dot. `""` will disable any truncation |
-| `format`   | `"[$hostname]($style) in "` | The format for the module.                                                                                                           |
-| `style`    | `"bold dimmed green"`       | The style for the module.                                                                                                            |
-| `disabled` | `false`                     | Disables the `hostname` module.                                                                                                      |
+| Option      | Default                     | Description                                                                                                                          |
+| ----------- | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| `ssh_only`  | `true`                      | Only show hostname when connected to an SSH session.                                                                                 |
+| `trim_at`   | `"."`                       | String that the hostname is cut off at, after the first match. `"."` will stop after the first dot. `""` will disable any truncation |
+| `format`    | `"[$hostname]($style) in "` | The format for the module.                                                                                                           |
+| `ssh_style` | `"bold yellow"`             | The style for remote hostname in SSH connection.                                                                                     |
```

Comment:
If I understand correctly, this also needs to be documented in the variables section. It also might be worth documenting that this is only used if an SSH connection is detected, otherwise strings styled with this variable will fall back to using `style`.